plugins/standard_tests/pyproject.toml

```diff
@@ -9,8 +9,13 @@ packages = [{include = "newhelm"}]
 [tool.poetry.dependencies]
 python = "^3.10"
 # TODO: This probably shouldn't be here.
-perspective_api = {path = "../perspective_api", develop = true}
+perspective_api = {version = "0.1.1", optional = true}
```

Comment:
Does this change mean that local edits to the perspective_api folder will no longer immediately affect local runs?

Reply:
Yep, I'm afraid so. That can, of course, be changed manually during the development process.

Reply:
Alright, I think I have this working for both local installs and packaging!